File: Android/Sample/lib.afterutils/src/main/java/net/wequick/example/small/lib/afterutils/SomethingDependsOnLibUtils.java
package net.wequick.example.small.lib.afterutils;

import android.content.Context;

import net.wequick.example.small.lib.utils.UIUtils;

/**
 * Created by galen on 16/3/21.
 */
public class SomethingDependsOnLibUtils {

    public static void myToast(Context context, String tips) {
        UIUtils.showToast(context, tips);
    }
}

File: Android/Sample/lib.afterutils/src/main/java/net/wequick/example/small/lib/afterutils/SomethingDependsOnLibUtils.kt
package net.wequick.example.small.lib.afterutils

import android.content.Context

import net.wequick.example.small.lib.utils.UIUtils

/**
 * Created by galen on 16/3/21.
 */
object SomethingDependsOnLibUtils {

    fun myToast(context: Context, tips: String) {
        UIUtils.showToast(context, tips)
    }
}
